paimon-web-ui-new/.eslintrc.cjs

```diff
@@ -0,0 +1,16 @@
+/* eslint-env node */
```

Comment:
Adds license for all cjs file. You could refer to [.eslintrc.cjs](https://github.com/apache/skywalking-booster-ui/blob/main/.eslintrc.cjs) of SkyWalking.